src/actions/WalletActions.js

```diff
@@ -245,10 +245,18 @@ export const getEnabledTokens = (walletId: string) => async (dispatch: Dispatch,
     }
     await Promise.all(promiseArray)
     // now reflect that change in Redux's version of the wallet
-    if (tokensToEnable.length) {
+    const plugins: Object = state.ui.settings.plugins
+    const allCurrencyInfos = plugins.allCurrencyInfos
+    // make sure that the token doesn't have same currencyCode
+    // as a parent plugin
+    const allSupportedParentCurrencies = allCurrencyInfos.map(info => info.currencyCode)
+    const filteredTokensToEnable = tokensToEnable.filter(token => {
+      return !allSupportedParentCurrencies.includes(token)
+    })
+    if (filteredTokensToEnable.length) {
```

Comment:
This block of code should have been moved into the loop above, assuming it was needed in the first place. It isn't needed, though, since `tokensToEnable` only includes currency codes that exist, and we have prevented tokens with conflicting currency codes from even existing to begin with. Therefore, the loop above already handles this case without doing anything extra.